wsclient/src/main/java/org/hpccsystems/ws/client/platform/Workunit.java

```diff
@@ -192,11 +192,8 @@ public static boolean isFailedState(String state) throws Exception
             WuStateNameMap.put("RUNNING", WUState.RUNNING);
     }
 
-    public static WUState translateWUState(String state) throws Exception
+    public static WUState translateWUState(String state)
     {
-        if (WuStateNameMap.size() <= 0)
```

Comment:
why was this block removed? If the static block above executes, we're never going to throw this exception. But if the code changes, this exception will let us quickly identify the issue.